Configuration/Test/test.py

```diff
@@ -0,0 +1,398 @@
+from SimG4Core.Application.hectorParameter_cfi import *
+import FWCore.ParameterSet.Config as cms
+
+#
+# 09.08.17 patrykel
+# used to test new optics in 8_1_0
+
+process = cms.Process("TestFlatGun")
+
+# Specify the maximum events to simulate
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(100)
+)
+
+# Configure the output module (save the result in a file)
+process.o1 = cms.OutputModule("PoolOutputModule",
+    outputCommands = cms.untracked.vstring('keep *'),
+    fileName = cms.untracked.string('file:test_100.root')
+)
+process.outpath = cms.EndPath(process.o1)
+
+################## STEP 1 - process.generator
+process.source = cms.Source("EmptySource")
+
+# particle generator paramteres
+process.load("Configuration.StandardSequences.Services_cff")
+
+process.load("Configuration.Generator.FlatLogKsiLogT_pp_13TeV_cfi")
+process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
+    g4SimHits = cms.PSet(initialSeed = cms.untracked.uint32(9876)),
+    SimG4Object = cms.PSet(initialSeed =cms.untracked.uint32(9876)),
+    RPSiDetDigitizer = cms.PSet(initialSeed =cms.untracked.uint32(137137)),
+    sourceSeed = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    generator = cms.PSet(initialSeed = cms.untracked.uint32(98766)),
+    SmearingGenerator = cms.PSet(initialSeed =cms.untracked.uint32(3849)),
+    T2MCl = cms.PSet(initialSeed =cms.untracked.uint32(24141)),
```

Comment:
T2MCl can be removed.

I'd suggest to load some generic `RandomNumberGeneratorService`. I think it exists somewhere in CMSSW repo.

Then use python list append or some other mechanism to extend it with TOTEM-like creatures. 
https://github.com/cms-sw/cmssw/blob/a5633ad784f98fe6d83a01a7d86e247a973ad60d/SimMuon/RPCDigitizer/python/rpcChamberMasker_cff.py#L38

I believe that TOTEM-specific values are only: RPSiDetDigitizer, LHCTransport.